src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -828,6 +828,23 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
   <data name="ConfigCommandKeyNotFound" xml:space="preserve">
     <value>Key '{0}' not found.</value>
   </data>
+  <data name="Error_ConfigSetInvalidKey" xml:space="preserve">
+    <value>'{0}' is not a valid config key.</value>
+    <comment>0 - The config key</comment>
+  </data>
+  <data name="ConfigUnsetCommandDescription" xml:space="preserve">
+    <value>Removes the key-value pair from a specified NuGet configuration setting.</value>
+  </data>
+  <data name="ConfigUnsetConfigFileDescription" xml:space="preserve">
+    <value>The NuGet configuration file from which settings will be removed.</value>
+  </data>
+  <data name="ConfigUnsetConfigKeyDescription" xml:space="preserve">
+    <value>The key of the settings that are to be removed.</value>
+  </data>
+  <data name="Error_ConfigUnsetNonExistingKey" xml:space="preserve">
+    <value>The config key "{0}" could not be unset.</value>
```

Comment:
Users might want to know why it could not be unset. If the use keys. How about `The key "{0}" could not be found in config section, so it could not be unset.`?